ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -151,21 +151,12 @@ def process_cluster_cli_params(config):
     # Importing here cause once the function is invoked we have already config
     # loaded, so this is OK to import once you sure that config is loaded.
     from ocs_ci.ocs.openshift_ops import OCP
-    if cluster_path:
-        OCP.set_kubeconfig(
-            os.path.join(cluster_path, ocsci_config.RUN['kubeconfig_location'])
-        )
-    # TODO: determine better place for parent dir
-    cluster_dir_parent = "/tmp"
-    default_cluster_name = ocsci_config.ENV_DATA.get('cluster_name', None)
+    OCP.set_kubeconfig(
+        os.path.join(cluster_path, ocsci_config.RUN['kubeconfig_location'])
+    )
     cluster_name = get_cli_param(config, 'cluster_name')
     if not cluster_name:
-        cluster_name = default_cluster_name
-    cid = random.randint(10000, 99999)
-    if not (cluster_name and cluster_path):
-        cluster_name = f"{cluster_name}-{cid}"
-    if not cluster_path:
-        cluster_path = os.path.join(cluster_dir_parent, cluster_name)
+        cluster_name = getuser()[:8]  # use first 8 chars of username
```

Comment:
I'm not sure if we can come up with something which will help us recognize the owner when `getuser()` fails, which makes me think that immediate failure would be a better outcome in such case. Timestamp is an extraneous information, as this information is already recorded for the cluster. IP address will be useful only when it's value is not in private range (eg. on a personal vm behind a nat) nor is dynamic one gathered from dhcp.

Reply:
there is a limit set by openshift instaler too, thats why we are only using 8 chars, anything extra(with prefix ocs-ci) is automatically truncated by installer.

Reply:
@vasukulkarni with this change we are only using the first 8 chars of the username, not prefixing with `ocs-ci-`. If we wanted to do that I can make that change, or I can up the char count to 15 since we don't need to compensate for it anymore. What do you think?

Reply:
@clacroix12 I think we can keep this as is before, add prefix ocs-ci + 8 chars, thanks